zrml/prediction-markets/src/orml_asset_registry.rs

```diff
@@ -0,0 +1,268 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
```

Comment:
This code is almost taken as it is from Centrifuge repo. AFAIU we need to keep their license note right?

Reply:
Can you explain a little bit, what this file does and what it is needed for? Thanks!

Reply:
No logical changes, just applied cargo fmt.